### DISCUSSION TITLE
Can't shut down pc

### DISCUSSION THREAD

**Discussion by JdogTheGreatOne** [AlexGustafsson/homebridge-wol#238] [Questions]
I tried shutting off my pc using homekit on my phone and it threw me this error. I'm pretty sure I've configured everything right on the plugin menu, any idea why this is happening? I also can't wake my pc either. The username and the password input is just my username and pin on the pc, right, not sure what else it would be?
![Screenshot 2023-03-23 015037](https://user-images.githubusercontent.com/113949225/227151249-348c545a-a73b-4fc4-a633-40912734c75f.png)

**Comment by AlexGustafsson** [comment]
This seems to be the same issue as in #241 - you're probably missing the `net` command.

**Reply by JdogTheGreatOne** [reply]
I have samba common bin installed, its the newest version. Do I need a wake command at all, because neither shutting down or waking is working. I have my pc on a static ip, and same with my raspberry pi. And the username and passcode between the % sign is my account username and the actual passcode for my windows account, not just the 4 digit pin i enter to get on at the start?

edit: WOL is now working! not sure what changed.
